components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -2576,7 +2584,7 @@ public void testIndexObjectbyRestrictedSystemUser() throws Exception {
     }
 
     /**
-     * Tests f it possible to modify a script using the upload method from
+     * Tests if it possible to modify a script using the upload method from
```

Comment:
Sorry, still missing "is" (or any verb in the sentence).
Suggestion 1:
``Test if it is possible to modify a script using the upload method from omero.client.``
Suggestion 2:
``Test if script can be modified using the upload method from omero.client.``

Reply:
Suggestion 3: 
``Test that a script cannot be modified using the upload method from omero.client.``